stdlib/database/sql/src/main/ballerina/sql/actions.bal

```diff
@@ -22,6 +22,7 @@ public type CallerActions object {
     #
     # + sqlQuery - The SQL stored procedure to execute
     # + recordType - Array of record types of the returned tables if there is any
+    # + parameters - The parameters to be passed to the procedure/function call. The number of parameters is variable
```

Comment:
"The number of parameters is variable" Is this sentence grammatically correct?

Reply:
Yes, variable is an adjective. (We use it as a noun as well)